app/src/Commands/Repo/UpdateCommand.php

```diff
@@ -33,16 +35,47 @@ public function __construct(RepoManager $repo_manager, Writer $writer)
     public function configure() : void
     {
         $this
-            ->addArgument("name", InputArgument::REQUIRED, "The name of the repository to be updated")
+            ->addArgument("name", InputArgument::OPTIONAL, "The name of the repository to be updated")
+            ->addOption("all", "a", InputOption::VALUE_NONE, "if is set all repos will be deleted")
             ->addOption("global", "g", InputOption::VALUE_NONE, "Determines if the repository to be updated is global")
         ;
     }
 
     public function execute(InputInterface $input, OutputInterface $output) : int
     {
         $name = $input->getArgument("name");
-        $global = $input->getOption("global");
+        $global = $input->getOption("global");$all = $input->getOption("all");
 
+        if (is_null($name) && ! $all) {
+            throw new InvalidArgumentException("Not enough arguments (missing: \"name\" or \"all\")");
+        }
+
+        if ($all) {
+            $repos = $this->repo_manager->getLocalRepos();
```

Comment:
Same as above, might not be needed if $global is true

Reply:
same as above